src/t2iapi/context/service.proto

```diff
@@ -105,6 +105,25 @@ service ContextService {
   t2iapi.context.CreateContextStateWithAssocAndSpecificValidatorRequest)
       returns (t2iapi.context.CreateContextStateWithAssociationResponse);
 
+  /*
+  Create or adapt a patient- or location context state, such that
+  - it belongs to the given @DescriptorHandle,
+  - it has @ContextAssociation = Assoc,
+  - it has the given number of pm:Validators, and
+  - its pm:Identification changes.
+  The pm:Identification of the created resp. adapted context state must differ structurally from
+  the pm:Identification of the previously associated context state of the same @DescriptorHandle.
+  In this context previous means directly prior to the manipulation call.
+  Returns the handle of the created or adapted context state and the mdib version
+  that contains the previously associated context state whose pm:Identification
+  the changed pm:Identification must structurally differ from.
```

Comment:
Instead of line 116 and as restriction which mdib version must be returned, pls add here the following sentence:

The mdib version must be the one that represents the mdib of the device before the changes of the manipulation are applied to the mdib.

Reply:
Done.